### PR TITLE
Added my profile (Ashutosh00710)

### PR DESCRIPTION
**Pull request by Ashutosh00710**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -244,24 +244,30 @@
...
         "imageUrl": "https://user-images.githubusercontent.com/31934448/95063061-77c76600-071b-11eb-813f-f53f9ba04035.png",
         "issuenumber": 128
-    }
+    },
+    {
+        "name": "Ashutosh Dwivedi",
+        "gitUsername": "Ashutosh00710",
+        "imageUrl": "https://user-images.githubusercontent.com/42907572/95647789-c3995700-0aef-11eb-97f3-01ff3b54cdaf.png",
+        "issuenumber": 134
+    },
```
Comment (eddiejaoude) [inline data.json:271]: Trailing comma is not needed on the last object

**Comment by eddiejaoude** [comment]
Thank you. I left 1 small comment

File: data.json (hunk 1 above)
Comment (Ashutosh00710) [reply data.json:271]: Okay sir I'll remove it.

Comment (Ashutosh00710) [reply data.json]: Sir have a look now the trailing comma has been removed

**Comment by Ashutosh00710** [comment]
@eddiejaoude  Sir please merge this pull request

**Comment by eddiejaoude** [comment]
I have resolved the conflict and I will now merge

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**